proc_Hahn_echo.py

```diff
@@ -20,11 +21,12 @@
     #fl.plot(s.imag,alpha=0.4)
     #fl.plot(abs(s),':',c='k',alpha=0.4)
     s.ft('t',shift=True)
-    fl.next('comp raw data - FT')
+    s.convolve('t',10)
+    fl.next('Water with Ni2+ - FT')
     #fl.plot(s.real,alpha=0.4)
     #fl.plot(s.imag,alpha=0.4)
     fl.plot(abs(s),c='red')
-    fl.next('comp raw data - FT')
+    fl.next('Water with Ni2+ - FT')
```

Comment:
but this is supposed to be code that processes Hahn echos, in general, so I don't understand why you're hard-coding such specific titles for the plots

Reply:
was for two years will fix though